### DISCUSSION TITLE
create the missing destination column if it's missing when uploading csv file [May 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2232] [Discord Threads]
> 🙋‍♂️ **Nils**  
## Is there a way that NocoDB just create the missing destination column if it's missing when I upload a .csv file?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/978644072447234098/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### for importing to an existing table (upload csv) - not possible at this moment    
#### for importing to an existing project (quick import) - it is possible  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/978644073432883240)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
